src/plugins/mavlink_ftp/mavlink_ftp_impl.h

```diff
@@ -0,0 +1,166 @@
+#pragma once
+
+#include <fstream>
+#include <mutex>
+#include <string>
+#include <dirent.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "mavlink_include.h"
+#include "plugins/mavlink_ftp/mavlink_ftp.h"
+#include "plugin_impl_base.h"
+
+namespace mavsdk {
+
+class MavlinkFTPImpl : public PluginImplBase {
+public:
+    /// @brief Command opcodes
+    enum Opcode : uint8_t {
```

Comment:
Please use an `enum class`.

Reply:
I need that enum values implicitly convert to uint8_t so that they're used according to protocol definition. They need to have the same values as PX4 Mavlink FTP server implementation. Is there a more elegant way to get around that?